workshops/geoserver/intro/doc/source/install/geoserver.rst

```diff
@@ -0,0 +1,213 @@
+.. install.geoserver:
+
+OSGeo Community GeoServer Distribution
+======================================
+
+The GeoServer project is part of the Open Source Geospatial Foundation. Two community releases are available at any one time:
+
+* stable: The latest production ready GeoServer along with optional extensions. New features and fixes are added to the stable release for the first six months.
+* maintenance: Fixes are added to maintenance releases for the next six months.
+
+GeoServer Comunity distributions include windows and mac installers along with standalone bin download. For those that already use Tomcat a web archive is alos available.

```

Comment:
Again, watch the caps please so we don't imply a product. Also, use "Windows" "macOS" and "binary download". And "also".